### PR TITLE
Test Updates & Schema Work

### PR DESCRIPTION
**Pull request by codenamejason**
- chore: get tests up and running and local setup

### CODE REVIEW

File: docker-compose.goerli.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - postgres
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    environment:
+      postgres_host: postgres
+      postgres_user: graph-node
+      postgres_pass: let-me-in
+      postgres_db: graph-node
+      ipfs: 'ipfs:5001'
+      ethereum: 'goerli:https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161'
```
Comment (0xKurt) [inline docker-compose.goerli.yml]: @codenamejason
is this your api key? I think we should remove it and use command line arguments instead?

File: docker-compose.pgn-testnet.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        "postgres",
+        "-cshared_preload_libraries=pg_stat_statements"
+      ]
+    environment:
+      POSTGRES_USER: graph-node
+      POSTGRES_PASSWORD: let-me-in
+      PGDATA: "/var/lib/postgresql/data"
+      POSTGRES_INITDB_ARGS: "-E UTF8 --locale=C"
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
```
Comment (thelostone-mc) [inline docker-compose.pgn-testnet.yml:45]: ^ let's remove this and hav a template file to generate these

File: config/goerli.json [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
-    "registryAddress": "0xAEc621EC8D9dE4B524f4864791171045d6BBBe27",
-    "alloAddress": "0x79536CC062EE8FAFA7A19a5fa07783BD7F792206"
-  }
\ No newline at end of file
+  "network": "goerli",
+  "startBlock": 9476300,
+  "registryAddress": "0xAEc621EC8D9dE4B524f4864791171045d6BBBe27",
+  "alloAddress": "0x79536CC062EE8FAFA7A19a5fa07783BD7F792206",
+  "donationVotingMerkleDistrobutionAddress": "0xC88612a4541A28c221F3d03b6Cf326dCFC557C4E",
+  "directGrantsSimpleStrategyAddress": "0xf243619f931c81617EE00bAAA5c5d97aCcC5af10"
```
Comment (thelostone-mc) [inline config/goerli.json]: let's move direct grants and donation voting to a different subgraph

File: src/core/allo.ts [hunk 4]
```diff
@@ -1,25 +1,28 @@
-import { Bytes, ByteArray, crypto, store, log } from '@graphprotocol/graph-ts';
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```
Comment (thelostone-mc) [inline src/core/allo.ts]: let's remove these

File: src/core/allo.ts [hunk 5]
```diff
@@ -1,25 +1,28 @@
...
+import { log, store } from '@graphprotocol/graph-ts';
 import {
+  BaseFeePaid,
```
Comment (thelostone-mc) [inline src/core/allo.ts]: Why do we need to track this ?

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**